core/src/core/transaction.rs

```diff
@@ -1366,20 +1366,12 @@ pub fn kernel_sig_msg(
 	if features.is_coinbase() && fee != 0 || !features.is_height_locked() && lock_height != 0 {
 		return Err(Error::InvalidKernelFeatures);
 	}
-	let msg = if global::is_floonet() {
-		let mut bytes = [0; 32];
-		BigEndian::write_u64(&mut bytes[16..24], fee);
-		BigEndian::write_u64(&mut bytes[24..], lock_height);
-		secp::Message::from_slice(&bytes)?
-	} else {
-		let hash = match features {
-			KernelFeatures::COINBASE => (features).hash(),
-			KernelFeatures::PLAIN => (features, fee).hash(),
-			_ => (features, fee, lock_height).hash(),
-		};
-		secp::Message::from_slice(&hash.as_bytes())?
+	let hash = match features {
```

Comment:
References to testnet4 in the function description comment should probably be removed as the corresponding code is being removed for the function body.

Reply:
Good catch, fixed.